foundation-ui/utils/ReactUtil.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ReactUil = {
+  /**
+   * Wrap React elements
+   *
+   * @param {Array.<ReactElement>|ReactElement} elements
+   * @param  {function|String} wrapper component
+   * @param {boolean} [alwaysWrap]
+   *
+   * @returns {ReactElement} wrapped react elements
+   */
+  wrapElements(elements, wrapper = 'div', alwaysWrap = false) {
```

Comment:
Good point! There are different ways to solve this; either we change the interface to something like the following or we keep it as it is and create custom wrapper components as illustrated below.

### Wapper Props 

Add the ability to pass on props to the wrapper

```
wrapElements(elements, wrapper = 'div', props = null, alwaysWrap = false)
```

### Custom Wapper 

Possible way using a custom wrapper to define a custom class name. 

```
wrapElements(elements, ({children}) => {<div className="custom-wrapper">{children}</div>});
```

What would you prefer?